bin/rucio-admin

```diff
@@ -2229,6 +2222,6 @@ if __name__ == '__main__':
         if args.verbose:
             print("Completed in %-0.4f sec." % (end_time - start_time))
         sys.exit(result)
-    except Exception as error:
-        logger.error("Strange error: {0}".format(error))
+    except Exception:
+        logger.exception("Uncaught exception")
```

Comment:
Should print the actual exception, no? Otherwise you have no clue what's actually failing?

Reply:
Or does .exception() do that automatically?

Reply:
exactly, `.exception` prints the exception with stacktrace while before there was no stacktrace and just the description of the exception as you can see in the issue.